webservices/filters.py

```diff
@@ -62,6 +62,19 @@ def filter_range(query, kwargs, fields):
     return query
 
 
+def filter_overlap(query, kwargs, fields):
+    for key, column in fields:
+        if kwargs.get(key):
+            # handle combination exclude/include lists
+            exclude_list = build_exclude_list(kwargs.get(key))
+            include_list = build_include_list(kwargs.get(key))
+            if exclude_list:
+                query = query.filter(~column.overlap(exclude_list))
+            if include_list:
+                query = query.filter(column.overlap(include_list))
+    return query
+
+
```

Comment:
If we're adding a new filter type, can you please add tests under `test_filters.py`?

Reply:
Per our discussion, more tests in test_committees.py is ok - I entered a follow up issue: https://github.com/fecgov/openFEC/issues/4674